docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -81,10 +81,7 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        %pip install --quiet yfinance
</pre><p>&gt;due to changes in yfinance i think</p><p> </p><p>Would you recommend requiring a certain version or higher of <code>yfianance</code> ? Like the current latest version or something </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
yep great point.

ending up moving the specific requirements (including versions) to gss/example-requirements to better align with what we do for css/qss